Source/WebCore/dom/TrustedType.cpp

```diff
@@ -173,6 +173,32 @@ ExceptionOr<String> trustedTypeCompliantString(TrustedType expectedType, ScriptE
     return stringValue;
 }
 
+ExceptionOr<String> trustedScriptCompliantStringFromVariant(ScriptExecutionContext& scriptExecutionContext, std::variant<RefPtr<TrustedScript>, String>&& input, const String& sink)
+{
+    return WTF::switchOn(
+        WTFMove(input),
+        [&scriptExecutionContext, &sink](const String& string) -> ExceptionOr<String> {
+            return trustedTypeCompliantString(TrustedType::TrustedScript, scriptExecutionContext, string, sink);
+        },
+        [](const RefPtr<TrustedScript>& script) -> ExceptionOr<String> {
+            return script->toString();
+        }
+    );
+}
+
+ExceptionOr<String> trustedScriptURLCompliantStringFromVariant(ScriptExecutionContext& scriptExecutionContext, std::variant<RefPtr<TrustedScriptURL>, String>&& input, const String& sink)
```

Comment:
This function doesn’t need `URL` or `FromVariant` in its name, because of how overloading works in C++. Can we just make it an overload for `trustedScriptCompliantString`?

Reply:
It takes a different type than TrustedScript though so I don't think it should be an overload of the other one. Unless I rename both to just trustedType... Rather than trustedScript.